CMakeLists.txt

```diff
@@ -18,6 +18,49 @@ option(ENABLE_DEVEL "Enable development asserts, checks, etc" OFF)
 option(ENABLE_REFCNT_DEBUG "Enable refcnt debugging" OFF)
 option(ENABLE_SHAREDPTR_DEBUG "Enable sharedptr debugging" OFF)
 set(TRYCOMPILE_SRC_DIR "${CMAKE_CURRENT_LIST_DIR}/packaging/cmake/try_compile")
+set(BUILT_WITH "")
+
+# Toolchain {
+if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
+  list(APPEND BUILT_WITH "GCC")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "Clang")
+  list(APPEND BUILT_WITH "CLANG")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+  list(APPEND BUILT_WITH "MSVC")
+endif()
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  list(APPEND BUILT_WITH "GXX")
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+  list(APPEND BUILT_WITH "CLANGXX")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+  list(APPEND BUILT_WITH "MSVCXX")
+endif()
+# }
+
+# GNU install {
+find_program(GnuInstall "install")
+if(NOT GnuInstall-NOTFOUND)
+  set(WITH_INSTALL ON)
+  list(APPEND BUILT_WITH "INSTALL")
+endif()
+# }
+
+# PkgConfig {
+find_package(PkgConfig QUIET)
```

Comment:
dito

Reply:
PkgConfig is used (if available on the host system) to configure SASL. So I would leave that in.